redos/7.3/Dockerfile

```diff
@@ -1,103 +1,32 @@
-FROM centos:7 as prepare
+FROM registry.red-soft.ru/ubi7/ubi-minimal:latest
```

Comment:
In the commit message you state that it is 7.3.3 version, but the tag is `latest`. It's very strange and absolutely wrong.

```console
curl -X GET https://registry.red-soft.ru/v2/ubi7/ubi-minimal/tags/list | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   369  100   369    0     0   3723      0 --:--:-- --:--:-- --:--:--  3727
{
  "name": "ubi7/ubi-minimal",
  "tags": [
    "7.3.2-230321",
    "7.3.3-230925",
    "7.3.2-230323",
    "7.3.2-230809",
    "7.3.2-230515",
    "7.3.3-230911",
    "7.3.2-230503",
    "7.3.2-230821",
    "7.3.2-221227",
    "7.3.2-230419",
    "7.3.2-230814",
    "7.3.2",
    "7.3.2-230731",
    "7.3.2-230411",
    "7.3.2-230712",
    "7.3.3-230904",
    "7.3.3-231002",
    "latest",
    "7.3.3-230918",
    "7.3.3-231009",
    "7.3.2-230608",
    "7.3.3-230926",
    "7.3.3-230828"
  ]
}
```

As you can see, `7.3.3-231009` is the latest 7.3.3 tag. Please use it.

Reply:
Fixed